tests/acceptance/pageObjects/FilesPageElement/SharingDialog/collaboratorsDialog.js

```diff
@@ -3,18 +3,123 @@ const util = require('util')
 module.exports = {
   commands: {
     /**
-     *
+     * @param {sting} collaborator
      * @returns {Promise.<string[]>} Array of autocomplete webElementIds
      */
-    deleteShareWithUserGroup: function (item) {
-      const informationSelector = util.format(this.elements.collaboratorInformationByCollaboratorName.selector, item)
+    deleteShareWithUserGroup: function (collaborator) {
+      const informationSelector = util.format(this.elements.collaboratorInformationByCollaboratorName.selector, collaborator)
```

Comment:
Please make sure you dont cross the vertical line on the screen.. Try to make lines shorter.